libs/blocks/section-metadata/sticky-section.js

```diff
@@ -24,9 +24,11 @@ function promoIntersectObserve(el, stickySectionEl, options = {}) {
 
 function handleStickyPromobar(section) {
   const main = document.querySelector('main');
+  section.classList.add('promo-sticky-section');
   section.classList.add('hide-sticky-section');
```

Comment:
```suggestion
```

Reply:
@ryanmparrish I have made the suggested change to remove the extra line.